.github/workflows/stack-regression-tests.yml

```diff
@@ -0,0 +1,77 @@
+name: stack-regression-tests
+on: 
+  pull_request:
+    branches:
+      - main
+jobs:
+   stack-tests:
+     runs-on: ubuntu-latest
+     steps:
+     
+      - name: Checkout
+        uses: actions/checkout@v2
+        
+      - name: Get stack image diff
+        id: stack-image-diff
+        uses: softprops/diffset@v1
+        with:
+          base: main
+          stackimage_files: |
+            stackimage/*
+            templates/stackimage/*
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} 
+          
+      - name: Get outer loop diff
+        id: outer-loop-diff
+        uses: softprops/diffset@v1
+        with:
+          base: main
+          outerloop_files: |
+            stackimage/*
+            templates/stackimage/*
+            templates/outer-loop/*
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} 
```

Comment:
Just for my own education, I'm curious:  what does this do?

Reply:
I'm honestly not sure... It's a "required" prop for the plugin, but it doesn't elaborate on it more that "GITHUB_TOKEN as provided by secrets"